openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -171,6 +171,9 @@ void isr_ieee154e_newSlot() {
    if (ieee154e_vars.isSync==FALSE) {
       if (idmanager_getIsDAGroot()==TRUE) {
          changeIsSync(TRUE);
+         incrementAsnOffset();

```

Comment:
Just for uniformity and to account for all timeslots elapsed: earlier, when the mote had detected to have been configured as a DAGroot (by openvisualizer) in a given timeslot, it would have changed to sync without incrementing the ASN in that timeslot. Now it does. So we can keep track of all the timeslots really elapsed from the beginning (the beginning is defined as ASN=0 on the mote that later has been configured as DAGroot).